### DISCUSSION TITLE
如何监听音频文件的播放进度

### DISCUSSION THREAD

**Discussion by a86356** [NervJS/taro#12140] [Q & A]
我的代码如下
`
 const innerAudioContext = Taro.createInnerAudioContext()
    innerAudioContext.autoplay = true
    innerAudioContext.src = url
    innerAudioContext.onPlay(() => {
      console.log('开始播放2312')
    })
    innerAudioContext.onError((res) => {
      console.log(res.errMsg)
      console.log(res.errCode)
    })
    innerAudioContext.onTimeUpdate=()=>{
      console.log(innerAudioContext.currentTime)
    }
`
我想实现的功能是监听音频的播放进度，但是onTimeUpdate这个方法没有触发，是要怎么弄呢？

**Comment by airqj** [comment]
onTimeUpdate相关的代码是不是写错了

**Reply by a86356** [reply]
没有吧 ，我按文档来的

**Reply by a86356** [reply]
哦哦哦，是为看错了，不好意思哈
